deployment/terraform/variables.tf

```
@@ -198,6 +198,8 @@ variable "google_server_side_api_key" {}
 
 variable "google_client_side_api_key" {}
 
+variable "google_analytics_key" {}
```

We need a default value here otherwise the `$ terraform plan` operation will be stuck at an interactive prompt. 

```suggestion
variable "google_analytics_key" {
  default = ""
}
```

---

Ah, so this is how defaults are specified, good to know! 👌